src/locale/locales/pt-BR/messages.po

```diff
@@ -2413,18 +2413,18 @@ msgstr "Todos"
 
 #: src/components/WhoCanReply.tsx:67
 msgid "Everybody can reply"
-msgstr ""
+msgstr "Todos podem responder"
 
 #: src/components/WhoCanReply.tsx:213
 msgid "Everybody can reply to this post."
-msgstr ""
+msgstr "Todos podem responder"
```

Comment:
missing end of sentence -> "Todos podem responder esta postagem"

Reply:
ok

Reply:
ok